_studio/shared/include/mfx_utils.h

```diff
@@ -99,9 +99,16 @@ mfxU64 GetMfxTimeStamp(mfxF64 ts)
 }
 
 inline
-bool LumaIsNull(mfxFrameSurface1 * surf)
+bool LumaIsNull(const mfxFrameSurface1 * surf)
```

Comment:
actually, here you also need a pointer check. But signature of this function doesn't allow to handle it natively, so I suggest to change signature of function to accept const reference to surface